setup.py

```diff
@@ -46,6 +46,33 @@
         'console_scripts': [
             'custodia = custodia.server:main'
         ],
+        'custodia.authenticators': [
+            'simplecreds = custodia.httpd.authenticators:SimpleCredsAuth',
```

Comment:
auth in simplecredsauth is redundant information. The plugin is qualified by its name space.